types/react-albus/index.d.ts

```diff
@@ -37,6 +37,7 @@ export interface WizardProps {
     render?: ((wizard: WizardContext) => React.ReactNode) | undefined;
     history?: History | undefined;
     basename?: string | undefined;
+    children?: NonNullable<React.ReactNode>;
```

Comment:
Why `NonNullable`?

And `React.ReactNode` doesn't account for children being a function as shown in https://github.com/americanexpress/react-albus/blob/v2.0.0/src/utils/renderCallback.js#L16-L21

Reply:
Hey - thanks for the review!

I believe that function is referencing something different - if you see where it's called [here](https://github.com/americanexpress/react-albus/blob/3d432cc0d3c3ac536269c0644a87ce61173eadcb/src/components/Wizard.js#L103).

I've chosen NonNullable for consistency with the children prop used in `Steps` - but I also think that's a correct decision, which is that you wouldn't pass `null` as a child to `<Wizard>`.

Happy to allow this to be any ReactNode though.

Reply:
`<Steps>` is different from `Wizard`, it does not use `renderCallback`, and if you follow the logic in its render method you'll see it only handles `ReactNode`.

`<Step>` on the other hand is like `Wizard` as it delegates rendering to `renderCallback`
- [Here how Step is created](https://github.com/americanexpress/react-albus/blob/main/src/index.js#L17)
- [Here Step calls renderCallback](https://github.com/americanexpress/react-albus/blob/3d432cc0d3c3ac536269c0644a87ce61173eadcb/src/components/createWizardComponent.js#L19)

So `renderCallback` defines which render/children values both `Step` and `Wizard` uses, which if you look at it is:
- render function via `render` prop
- render function via `children` prop
- ReactNode via `children` prop

eg. the `WizardContextRenderProps`.


Therefore, I think your change should be:
```ts
export interface WizardProps extends WizardContextRenderProps {
  ...
}
```

And, if you look at the props of `Step`, you'll see they are similarly defined and working as expected:
```ts
export type StepProps = StepObject & WizardContextRenderProps;
```

Reply:
Thanks so much - I've done this now :)